app/helpers/orders_helper.rb

```diff
@@ -1,2 +1,13 @@
 module OrdersHelper
+    def send_notification (order)
```

Comment:
Nagyon teszik ez a megoldás, igazán hasznos lesz szerintem.
Pár dolog kéne bele:
-  Csak production környezetben küldjünk notificationt, hogy amikor localban tesztelünk, akkor nem zavarjunk be
- Hibakezelés: ha most ledjuk a rendelést és valami hiba van a notification-ben, akkor a user 500-at kap, nem tudja, hogy amúgy beküldte a rendelést, csak a notival volt valami gond, mondjuk timeoutolt a ntfy

Reply:
Nem tudom ez így mennyire etikus hibakezelésnek. Úgy tudom, hogy a `Net::HTTP` egyszer fog próbálkozni, bármilyen error esetén és ha ezek után is hibát dob, akkor simán visszatér a függvény.